### PR TITLE
[LW-9151] Edit account name component

### PR DESCRIPTION
**Pull request by lucas-barros**
# Checklist

- [x] JIRA - https://input-output.atlassian.net/browse/LW-9151
- [x] Proper tests implemented
- [x] Screenshots added.

---

## Proposed solution

Explain how does this PR solves the problem stated in JIRA ticket.
You can also enumerate different alternatives considered while approaching this task.

## Testing

Describe here, how the new implementation can be tested.
Provide link or briefly describe User Acceptance Criteria/Tests that need to be met

## Screenshots

image

image

### CODE REVIEW

File: packages/ui/src/design-system/input/input.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+import { Input } from './input.component';
+import * as cx from './input.stories.css';
+
+export default {
+  title: 'Input Fields/Input',
+  component: Input,
+  decorators: [
+    page({
+      title: 'Input',
```
Comment (renanvalentin) [inline packages/ui/src/design-system/input/input.stories.tsx:19]: We are missing the component description.

File: apps/browser-extension-wallet/src/features/account/components/EditAccount/EditAccountDrawer.test.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { render, fireEvent, screen } from '@testing-library/react';
+import { EditAccountDrawer } from './EditAccountDrawer';
+import '@testing-library/jest-dom';
+
+jest.mock('react-i18next', () => ({
+  useTranslation: () => ({ t: jest.fn() })
+}));
+
+describe('EditAccountDrawer', () => {
```
Comment (DominikGuzei) [inline apps/browser-extension-wallet/src/features/account/components/EditAccount/EditAccountDrawer.test.tsx:10]: Cool stuff and are we sure that we want to write these tests here when they probably gonna be covered in a similar way by the SDET team with e2e tests?

**Comment by DominikGuzei** [comment]
The input components story seems to be broken on smaller screen sizes:
image

File: packages/ui/src/design-system/input/index.ts [hunk 3]
```diff
@@ -0,0 +1,2 @@
+export { Input } from './input.component';
+export type { InputProps } from './input.component';
```
[the remark this answers is not in the document]
Comment (lucas-barros) [reply packages/ui/src/design-system/input/index.ts]: Addressed here: https://github.com/input-output-hk/lace/pull/746/commits/ad6653ff45434e3d1be94912a39cb08d430ccdd8. Thanks!

File: packages/ui/src/design-system/input/input.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+  ],
+} as Meta;
+
+const MainComponents = (): JSX.Element => (
+  <Variants.Row>
+    <Variants.Cell>
+      <Input label="Label" value="" />
+    </Variants.Cell>
+    <Variants.Cell>
+      <Input label="Label" value="" containerClassName={cx.hoverEffect} />
```
[the remark this answers is not in the document]
Comment (lucas-barros) [reply packages/ui/src/design-system/input/input.stories.tsx]: Addressed here: https://github.com/input-output-hk/lace/pull/746/commits/8dee4ad0e6ea048775f0985405c619b7becd66e2. Thanks!

File: packages/ui/src/design-system/input/input.stories.tsx (hunk 1 above)
Comment (lucas-barros) [reply packages/ui/src/design-system/input/input.stories.tsx]: Addressed here: https://github.com/input-output-hk/lace/pull/746/commits/8dee4ad0e6ea048775f0985405c619b7becd66e2. Thanks!
Although myb description is not the best, feel free to add a sugestion!

File: apps/browser-extension-wallet/src/features/account/components/EditAccount/EditAccountDrawer.test.tsx (hunk 2 above)
Comment (lucas-barros) [reply apps/browser-extension-wallet/src/features/account/components/EditAccount/EditAccountDrawer.test.tsx:10]: I think they are valid as it's quicker feedback for developers when updating the component, but I'm opened for a deeper discussion.

**Comment by lucas-barros** [comment]
> The input components story seems to be broken on smaller screen sizes: image

@DominikGuzei fixed: https://github.com/input-output-hk/lace/pull/746/commits/02a5ebe2a8911303355b0ae97606b76c62304219

**Review by DominikGuzei — APPROVED** [review]
Great work 🎉  I saw that the `yarn.lock` file might not be up-to-date since it showed changes when i ran `yarn install` locally (only adding `@lace/ui`)

**Review by renanvalentin — APPROVED** [review]
great job @lucas-barros !

**Pull request merged**